### PR TITLE
Release 3.0.0

### PR DESCRIPTION
**Pull request by frankheijnders**
Wijzigingen om Documentation als ZIP (en als tar.gz) pakketje uit te spugen.

### CODE REVIEW

**Comment by CLAAdmin** [comment]
[CLA assistant check](http://52.16.237.165:80/OSGP/Documentation?pullRequest=33) <br/>You should sign our Contributor License Agreement in order to get your pull request merged.<br/>

File: .gitignore [hunk 1]
```diff
@@ -1,16 +1 @@
...
-node_modules
-
-# Book build output
-_book
-
-# eBook build output
-*.epub
-*.mobi
-*.pdf
+target/
```
Comment (kevinsmeets) [inline .gitignore]: The gitignore file was fine i think, if you need to add 'target/' to the file you can do so. However, deleting every gitignore rule is not OK. Please restore the gitignore file and add 'target/' to it.

File: assembly-plugin-config.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<assembly
```
Comment (kevinsmeets) [inline assembly-plugin-config.xml]: Please add a header to this file.

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,78 @@
+<!--
+
+    Copyright 2015 Smart Society Services B.V.
+
+    Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (kevinsmeets) [inline pom.xml]: Please format the XML using 2 spaces, no tabs. Make sure to keep the header as is.

File: .gitignore (hunk 1 above)
Comment (frankheijnders) [inline .gitignore]: Fixed

File: assembly-plugin-config.xml (hunk 2 above)
Comment (frankheijnders) [inline assembly-plugin-config.xml]: Copied header from POM

File: pom.xml (hunk 3 above)
Comment (frankheijnders) [inline pom.xml]: Fixed indentation. Hopefully header is now as it was?

**Comment by kevinsmeets** [comment]
OK

**Pull request merged**